packages/server/src/email.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import email from 'emailjs';
+
+class Email {
+  constructor(emailConfig: Object) {
+    if (emailConfig) {
+      this.server = email.server.connect(emailConfig);
+    }
+  }
+
+  sendMail(mail: Object): Promise<Object> {
+    return new Promise((resolve, reject) => { // eslint-disable-line flowtype/require-parameter-type
+      // If no configuration for email just warm the user
+      if (!this.server) {
+        console.log('No configuration for email, you must set an email configuration');
+        resolve();
+        return;
+      }
+      this.server.send(mail, (err: Object, message: Object) => {
```

Comment:
Would be cool if user could set a function in the config to handle sending emails. At the moment I think this only supports smtp, but what about the use case when you want to do an api call to mailgun to send the email?